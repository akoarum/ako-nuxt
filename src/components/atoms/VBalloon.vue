<template>
  <div class="vBalloon">
    <slot />
  </div>
</template>

<script>
export default {}
</script>

<style lang="scss" scoped>
.vBalloon {
  border-radius: 2px;
  background: rgba(0, 0, 0, 0.72);
  color: #fff;

  @include media-min {
    padding: 8px 16px;
  }
}
</style>
