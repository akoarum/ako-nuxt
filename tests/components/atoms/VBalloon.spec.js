import { shallowMount } from '@vue/test-utils'
import VBalloon from '~/components/atoms/VBalloon'

describe('VBalloon', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(VBalloon, {
      propsData: {}
    })
  })

  describe('initialized', () => {
    it('mountable', () => {
      expect(wrapper.isVueInstance()).toBe(true)
    })
    it('snapshot', () => {
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
