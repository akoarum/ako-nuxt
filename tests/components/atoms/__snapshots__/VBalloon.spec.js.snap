// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VBalloon initialized snapshot 1`] = `
<div
  class="vBalloon"
/>
`;
